jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -73,6 +73,10 @@ public void setEntrypoint(List<String> entrypoint) {
     this.entrypoint = entrypoint;
   }
 
+  public void setEntrypoint(String entrypoint) {
```

Comment:
Maybe this doesn't matter, but `INHERIT` and [`INHERIT`] will both trigger the keyword path?

Reply:
Yeah, that's the way it works, which I thought OK. But if you do think `['INHERIT']` shouldn't inherit, I can block it, but I think that'll require larger code changes.